prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -115,7 +124,7 @@ class FHIRRouter(
      */
     val processingModeDefaults = mapOf(
         Pair(Topic.FULL_ELR, processingModeFilterDefault),
-        Pair(Topic.ETOR_TI, processingModeFilterDefault),
+        Pair(Topic.ETOR_TI, processingModeFilterDefault)
```

Comment:
I want to make a pitch in favor of changing your formatter settings to allow trailing commas, but not important 😄 Also want to mention that I'm happy with not having a default proc mode filter for the new topic here for now.